README.md

```diff
@@ -67,137 +39,173 @@ Transfer/sec:     68.37MB
 
 ## Installation
 
-Add this to your application's `shard.yml`:
+#### Linux
 
-```yaml
-dependencies:
-  amber:
-    github: Amber-Crystal/amber
-```
+Ensure you have the necessary dependencies:
 
-## Usage
+- `git`: Use your platform specific package manager to install `git`
+- `crystal`: Follow the instructions to get `crystal` on this page: <https://crystal-lang.org/docs/installation/index.html>
 
-Please see [Installing Amber CLI from the Amber Book](https://amber-crystal.gitbooks.io/amber/content/getting-started/Installation/)
+##### For Debian & Ubuntu
+- `sudo apt-get build-essential libreadline-dev libsqlite3-dev libpq-dev libmysqlclient-dev libssl-dev`
```

Comment:
Is `libsqlite3-dev` really required? That may trigger errors during deployment to Heroku, for example...

Reply:
It's not required to run a server but it is require to compile the amber cli.